src/main/java/io/strimzi/kafka/bridge/NoopPartitionsRebalance.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge;
+
+import org.apache.kafka.clients.consumer.ConsumerRebalanceListener;
+import org.apache.kafka.common.TopicPartition;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collection;
+
+/**
+ * No operations implementation about handling partitions being assigned on revoked on rebalancing
+ * It just logs partitions if enabled
+ */
+public class NoopPartitionsRebalance implements ConsumerRebalanceListener {
```

Comment:
Should we call it `LoggingPartitionsRebalance`, since that's what it does?

Reply:
I was wandering the same, but waiting for feedback on my feeling without spoilering it :-) I will change the name.